ivy/functional/frontends/torch/Tensor.py

```diff
@@ -66,7 +66,10 @@ def amax(self, dim=None, keepdim=False):
     def contiguous(self, memory_format=torch.contiguous_format):
         return self.data
 
-    # Special Methoods #
+    def max(self, dim=None, keepdim=False):
+        return torch_frontend.max(self.data, dim=dim, keepdim=keepdim)
```

Comment:
Hi 🙂
The torch frontend doesn't have a `max()` function yet.
This is an instance method that you've implemented, which should call the ordinary `max()` function for torch frontend which is not implemented yet.
 
We'd love if you'd implement the ordinary function too, as every contribution here at Ivy is appreciated.
Thanks! 🙂